### ISSUE TITLE
blockedByDefault is ignored?

### ISSUE THREAD

**Issue by FezVrasta** [airbnb/react-dates#134]
So... I'm using `SingleDatePicker` and I'd like to make selectable only the array of Moment dates I pass as `unblockedDate`

```
          <SingleDatePicker
            id="datePicker"
            onDateChange={this.onDateChange}
            onFocusChange={this.onFocusChange}
            focused={focused}
            date={date}
            blockedByDefault
            unblockedDates={unblockedDates}
            numberOfMonths={1}
          />
```

But when I render it, all the future dates are selectable even if in my `unblockedDates` there's only a single Moment date.

Looking in the code base I can't find any reference to `blockedByDefault` 🤔

Why?

**Comment by FezVrasta** [comment]
As workaround I used `isDayBlocked` checking inside it if the day should be blocked or not... But `blockedByDefault` would have been much better

**Comment by majapw** [comment]
Hi @FezVrasta, we deprecated `blockedByDefault`, `blockedDates`, and `unblockedDates` in favor of `isDayBlocked` in https://github.com/airbnb/react-dates/pull/4 since it seemed much simpler to have one prop instead of three. Unfortunately, it seems I forgot to update the README at that time. I will update the `README`.

tl;dr - `isDayBlocked` is not a workaround; it is the recommended solution.

**Comment by FezVrasta** [comment]
Thanks 😊
